Formula/genstats.rb

```diff
@@ -1,8 +1,12 @@
 class Genstats < Formula
   desc "Generate statistics about stdin or textfiles"
-  homepage "https://www.vanheusden.com/genstats/"
-  url "https://www.vanheusden.com/genstats/genstats-1.2.tgz"
-  sha256 "f0fb9f29750cdaa85dba648709110c0bc80988dd6a98dd18a53169473aaa6ad3"
+  homepage "https://web.archive.org/web/20180831170016/www.vanheusden.com/genstats/"
+  url "https://web.archive.org/web/20150331055106if_/vanheusden.com/genstats/genstats-1.2.tgz"
+  sha256 "9988264357211a24f7024db05e24ed88db58227a626330114309147eb7078f6e"
+  license "GPL-2.0"
+  livecheck do
+    skip "Upstream is gone and the formula uses archive.org URLs"
+  end
```

Comment:
Shouldn't this work?
```suggestion
```

Reply:
Just following suit of what xml2 ~~does~~ did, which was removed in https://github.com/Homebrew/homebrew-core/commit/2cfebaa790bb5e11581810e7170535b2265bf1bc. I'll remove it.